iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/AbstractTapChanger.java

```diff
@@ -24,7 +24,7 @@
 
     protected final H parent;
 
-    protected int lowTapPosition;
+    protected final int lowTapPosition;
```

Comment:
To be discussed: final vs setter

Reply:
Add a setter: be careful you have to ensure the consistency between lowTapPosition and tapPosition. After changing the lowTapPosition, the tapPosition must reference the same tap.

I think it's not necessary to notify any change of the tapPosition in that case.

A possible implementation is to compute the distance between the new and the old values and to increment lowTapPosition with it:
```
int oldValue = this.lowTapPosition;
this.lowTapPosition = lowTapPosition;
// notify
this.tapPosition += (this.lowTapPosition - oldValue);
```